Mathlib/MeasureTheory/Function/SpecialFunctions/Sinc.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2025 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Sinc
+import Mathlib.MeasureTheory.Function.SpecialFunctions.Basic
+import Mathlib.MeasureTheory.Function.L1Space.Integrable
+
+/-!
+# Measurability and integrability of the sinc function
+
+## Main statements
+
+* `measurable_sinc`: the sinc function is measurable.
+* `integrable_sinc`: the sinc function is integrable with respect to any finite measure on `ℝ`.
+
```

Comment:
Maybe you can add some TODOs for the Taylor series representation, analyticity, etc.